### ISSUE TITLE
how to test trained models to generate scanpath?

### ISSUE THREAD

**Issue by zhangkp1997** [cvlab-stonybrook/Scanpath_Prediction#19]

**Comment by ouyangzhibo** [comment]
For now, you may evaluate the model using the validation set. Also, feel free to contact Greg (gregory.zelinsky@stonybrook.edu) to request the access to the test set.
